eve/workers/openstack-multiple-nodes/terraform/nodes.tf

```diff
@@ -92,7 +92,10 @@ resource "openstack_compute_instance_v2" "bootstrap" {
 
   # Generate BootstrapConfiguration
   provisioner "remote-exec" {
-    inline = ["sudo bash scripts/bootstrap-config.sh"]
+    inline = [
+      "sudo chmod +x scripts/bootstrap-config.sh",
```

Comment:
Can't we simply make this script executable directly on the repository, so we don't need to `chmod` it each time.
I don't know if the file provisioner keep the mode of a file, but I think it is.

Reply:
Done